app/shops.py

```diff
@@ -0,0 +1,5 @@
+class Shop:
+    def __init__(self, shops_dict: dict) -> None:
```

Comment:
This way of class initialization is too ambiguous, pass directly arguments that will be used in the attributes assignment. Additionally, you could consider this class being a dataclass. The same comments relate to the `Customer` class as well.